src/config.c

```diff
@@ -46,30 +46,29 @@ configPair_t __configPairs[] = {
   {"FORK_GC_CLEAN_THRESHOLD",         "search-fork-gc-clean-threshold"},
   {"FORK_GC_RETRY_INTERVAL",          "search-fork-gc-retry-interval"},
   {"FORK_GC_RUN_INTERVAL",            "search-fork-gc-run-interval"},
-  {"FORKGC_SLEEP_BEFORE_EXIT",        "search-forkgc-sleep-before-exit"},
+  {"FORKGC_SLEEP_BEFORE_EXIT",        "search-fork-gc-sleep-before-exit"},
   {"FRISOINI",                        "search-friso-ini"},
-  {"GC_POLICY",                       "search-gc-policy"},
```

Comment:
Need to adjust the deprecation message when there's no new config parameter replacing the old config parameter

Reply:
We already have a warning
> RedisModule_Log(ctx, "warning", "FT.CONFIG is deprecated, please use CONFIG instead");

But specifying the corresponding new `CONFIG` name seems better.
I don't think we need both warnings in case a change was done.